projects/plugins/jetpack/extensions/blocks/ai-assistant/upgrade-prompt.js

```diff
@@ -19,7 +19,10 @@ const UpgradePrompt = () => {
 			buttonText={ 'Upgrade' }
 			checkoutUrl={ `${ window?.Jetpack_Editor_Initial_State?.adminUrl }admin.php?page=my-jetpack#/add-jetpack-ai` }
 			className={ 'jetpack-ai-upgrade-banner' }
-			description={ __( 'Free requests used. Upgrade now to keep using Jetpack AI.', 'jetpack' ) }
+			description={ __(
+				'You have reached the limit of 20 free requests. Upgrade now to keep using Jetpack AI.',
```

Comment:
Just a suggestion for a generic message, still friendly but flexible:

```suggestion
				'You have reached the limit of free requests. Upgrade now to keep using Jetpack AI.',
```